components/automate-ui/src/app/modules/team/team-details/team-details.component.ts

```diff
@@ -133,27 +126,11 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     ).subscribe((team) => {
       this.teamId = team.id;
       this.team = team;
-      this.updateNameForm.controls.name.setValue(this.team.name);
+      this.updateForm.controls.name.setValue(this.team.name);
       this.store.dispatch(new GetTeamUsers({ id: this.teamId }));
       this.store.dispatch(new GetProjects());
     });
 
-    combineLatest([
```

Comment:
Entire subscription was just to populate `this.projects` which has now been completely excised.